bundles/org.openhab.binding.unifiprotect/src/main/java/org/openhab/binding/unifiprotect/internal/media/UnifiMediaServiceImpl.java

```diff
@@ -0,0 +1,321 @@
+/*
+ * Copyright (c) 2010-2025 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.unifiprotect.internal.media;
+
+import java.io.IOException;
+import java.net.URI;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.stream.Collectors;
+
+import javax.servlet.ServletException;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.jdt.annotation.Nullable;
+import org.eclipse.jetty.client.HttpClient;
+import org.eclipse.jetty.util.ssl.SslContextFactory;
+import org.openhab.binding.unifiprotect.internal.UnifiProtectBindingConstants;
+import org.openhab.binding.unifiprotect.internal.config.UnifiProtectConfiguration;
+import org.openhab.binding.unifiprotect.internal.handler.UnifiProtectCameraHandler;
+import org.openhab.core.OpenHAB;
+import org.openhab.core.config.core.ConfigurableService;
+import org.openhab.core.config.core.Configuration;
+import org.openhab.core.io.net.http.HttpClientFactory;
+import org.openhab.core.thing.ThingUID;
+import org.osgi.framework.Constants;
+import org.osgi.service.component.annotations.Activate;
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.Deactivate;
+import org.osgi.service.component.annotations.Modified;
+import org.osgi.service.component.annotations.Reference;
+import org.osgi.service.http.HttpService;
+import org.osgi.service.http.NamespaceException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Implementation of the WebRtcMediaService interface.
+ *
+ * @author Dan Cunningham - Initial contribution
+ */
+@Component(service = UnifiMediaService.class, immediate = true, configurationPid = UnifiProtectBindingConstants.SERVICE_ID, property = Constants.SERVICE_PID
+        + "=" + UnifiProtectBindingConstants.SERVICE_ID)
+@ConfigurableService(category = "system", label = "UnifiProtect", description_uri = "binding:unifiprotect")
```

Comment:
This is putting the configuration under System Settings rather than Add-On Settings as one would expect.

EDIT: Actually, if I expand it to the right, it shows up there, too (and with the same configuration settings)! That's definitely confusing.

<img width="742" height="593" alt="image" src="https://github.com/user-attachments/assets/c6f9b80a-cac0-4a82-815c-19d122aac9ff" />

Reply:
ah, good catch !